e2e/test/scenarios/dashboard-filters/reproductions/29997-multiple-values-parameters-with-template-tags.cy.spec.js

```diff
@@ -0,0 +1,139 @@
+import {
+  restore,
+  editDashboard,
+  visitDashboard,
+  getDashboardCard,
+  selectDashboardFilter,
+  filterWidget,
+  popover,
+} from "e2e/support/helpers";
+
+import * as FieldFilter from "e2e/test/scenarios/native-filters/helpers/e2e-field-filter-helpers";
+
+const nativeQuestionDetails = {
+  name: "Native question with state parameter",
+  native: {
+    query: "select * from people where state = {{state}};",
+    "template-tags": {
+      state: {
+        type: "text",
+        name: "state",
+        id: "c2efec27-91b7-4c2d-b584-c07c759c554e",
+        "display-name": "State",
+      },
+    },
+  },
+};
+
+const nativeQuestionParameter = {
+  id: "c2efec27-91b7-4c2d-b584-c07c759c554e",
+  name: "State",
+  slug: "state",
+  target: ["variable", ["template-tag", "state"]],
+  type: "category",
+  values_query_type: "list",
+};
+
+const dashboardParameter = {
+  name: "State",
+  slug: "state",
+  id: "f8ae0c97",
+  type: "string/=",
+  sectionId: "location",
+  values_query_type: "list",
+};
+
+const customValues = {
+  values_source_config: { values: ["AK", "AL", "AR", "AZ", "CA"] },
+  values_source_type: "static-list",
+};
+
+const dashboardDetails = {
+  name: "Multiple values parameter with template tags",
+};
+
+describe("issue 29997", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it('should not see "people can pick" option when connecting a native questions to a dashboard parameter with custom values (metabase#29997)', () => {
```

Comment:
Adding this scenario where we connect
1. A native question with a template tag **without** custom values to
2. A dashboard parameter with custom values

Since this is the only scenario I can reproduce the original issue's photo
<img src="https://user-images.githubusercontent.com/1711649/231255642-afdd6625-8315-4feb-b2f1-1b156660330c.png" />

where the dashboard parameter shows a list of options.